.github/workflows/ci.yml

```diff
@@ -327,6 +325,26 @@ jobs:
           python make.py python-ci-install
         timeout-minutes: 20
 
+      # We only save the libparsec lib when:
+      # - We are not in a github queue branch (they're a one time use so caching won't help)
+      # - The rust code isn't modifed.
```

Comment:
isn't it the opposite: we want  to save libparsec in cache because the rust code has been modified (and hence libparsec has been rebuilt from the sources) ?

Reply:
https://github.com/Scille/parsec-cloud/blob/2fc9e47d7dd8cbf2e0611ad1988d48a370d08467/.github/workflows/ci.yml#L280-L281

I've a comment made by you that said otherwise ;)

Here is the following condition to meet:

- On a PR, if we change the rust code, we could not reuse the pre-compiled lib.
- On the merge queue, we never want to save the compiled lib (the branch is a one-time use)
- On master, we want to save the pre-compile lib for later PR

Reply:
Continue the discussion on https://github.com/Scille/parsec-cloud/pull/4234#discussion_r1138424381